recipes/client.rb

```diff
@@ -62,9 +62,9 @@
   owner 'root'
   group 'root'
   variables(
-    mac: SshMac.get_macs(node, node['ssh']['client']['weak_hmac']),
-    kex: SshKex.get_kexs(node, node['ssh']['client']['weak_kex']),
-    cipher: SshCipher.get_ciphers(node, node['ssh']['client']['cbc_required']),
+    mac: DevSec.get_client_macs(node, node['ssh']['client']['weak_hmac']),
```

Comment:
Should we use `DevSec::SSH` as namespace? In order to avoid name conflicts with other cookbooks?